### ISSUE TITLE
Is the COCOA-cls result trained from scratch or COCO pretrained?

### ISSUE THREAD

**Issue by zbwxp** [UARK-AICV/AISFormer#1]
Hi @trqminh 

Really nice work done~

Just want to confirm that is the COCOA-cls result trained from scratch using resnet50 pretrain only?
Or use the Mask-RCNN weights trained on COCO dataset then finetune on COCOA-cls?

Thank you very much

**Comment by trqminh** [comment]
Hi @zbwxp ,
Thank you for your interest in our work and sorry for the late response. 
We used Mask-RCNN weights trained on COCO dataset then finetuned on COCOA-cls.
